data/os/RedHat.yaml

```diff
@@ -12,7 +12,7 @@ bacula::director::services: 'bacula-dir'
 bacula::storage::services: 'bacula-sd'
 bacula::client::services: 'bacula-fd'
 bacula::conf_dir: '/etc/bacula'
-bacula_dir: '/etc/bacula/ssl'
+bacula::bacula_dir: '/etc/bacula/ssl'
```

Comment:
I think this can actually be used.  I believe that variable has been replaced by `bacula::conf_dir` above this one.